.ado/jobs/universal.yml

```diff
@@ -1,147 +1,227 @@
+                      Mso.UnitTests/Mso.UnitTests.exe
+                    pathtoCustomTestAdapters: $(GoogleTestAdapterPath)
+                    searchFolder: $(Build.SourcesDirectory)/vnext/target/${{ matrix.BuildPlatform }}/${{ matrix.BuildConfiguration }}
+                    runTestsInIsolation: true
+                    platform: ${{ matrix.BuildPlatform }}
+                    configuration: ${{ matrix.BuildConfiguration }}
+                    publishRunAttachments: true
+                    collectDumpOn: onAbortOnly
+                    vsTestVersion: latest
+                  condition: and(succeeded(), not(startsWith(variables.BuildPlatform, 'arm')))
+
+                - task: VSTest@2
+                  displayName: Run Universal Unit Tests (UWP)
+                  timeoutInMinutes: 5 # Set smaller timeout , due to hangs
+                  inputs:
+                    testSelector: testAssemblies
+                    testAssemblyVer2: |
+                      Microsoft.ReactNative.Managed.UnitTests/Microsoft.ReactNative.Managed.UnitTests/Microsoft.ReactNative.Managed.UnitTests.build.appxrecipe
+                    searchFolder: $(Build.SourcesDirectory)/vnext/target/${{ matrix.BuildPlatform }}/${{ matrix.BuildConfiguration }}
+                    runTestsInIsolation: true
+                    platform: ${{ matrix.BuildPlatform }}
+                    configuration: ${{ matrix.BuildConfiguration }}
+                    publishRunAttachments: true
+                    codeCoverageEnabled: true
+                    collectDumpOn: onAbortOnly
+                    vsTestVersion: latest
+                  condition: and(succeeded(), not(startsWith(variables.BuildPlatform, 'arm')))
+
+                - task: VSTest@2
+                  displayName: Run Universal Unit Tests (NetCore)
+                  timeoutInMinutes: 5 # Set smaller timeout , due to hangs
+                  inputs:
+                    testSelector: testAssemblies
+                    testAssemblyVer2: |
+                      Microsoft.ReactNative.Managed.CodeGen.UnitTests/Microsoft.ReactNative.Managed.CodeGen.UnitTests.dll
+                    searchFolder: $(Build.SourcesDirectory)/vnext/target/${{ matrix.BuildPlatform }}/${{ matrix.BuildConfiguration }}
+                    runTestsInIsolation: true
+                    platform: ${{ matrix.BuildPlatform }}
+                    configuration: ${{ matrix.BuildConfiguration }}
+                    publishRunAttachments: true
+                    codeCoverageEnabled: true
+                    collectDumpOn: onAbortOnly
+                    vsTestVersion: latest
+                  condition: and(succeeded(), eq(variables.BuildPlatform, 'x64'))
+
+    # This job is the one that accumulates the spread out build tasks into one dependency
+    - job: UniversalBuild
+      condition: ne( dependencies.Setup.outputs['checkPayload.shouldSkipPRBuild'], 'True' )
+      displayName: Universal Build 🏗
```

Comment:
It is absolutely blowing my mind that we can put emojis in display names. We need more...

Reply:
Was just experimenting. I'll put more in a separate PR, already have a list of candidates :P